src/internal/markers.ts

```diff
@@ -176,9 +176,8 @@ export const findMarker = (observable$: Observable<unknown>): Marker | null => {
     return observable$.marker;
   } else if (observable$.source instanceof Observable) {
     return findMarker(observable$.source);
-  } else {
-    return null;
   }
+  return null;
```

Comment:
Redundant `else` after a `return` in `if` (`no-else-return`). I personally think we should use this rule.
```
if (thing) {
  return 1;
} // no else here
return 2;
```